packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js

```diff
@@ -67,14 +68,13 @@ export default function InputDatePicker(props) {
 	return (
 		<Date.Manager
 			value={props.value}
-			textInput={props.textInput}
```

Comment:
we don't need textInput passed, because value of date input is only determined by `props.value`